dev_requirements.txt

```diff
@@ -1,5 +1,6 @@
 click==8.0.4
 black==24.3.0
+cachetools
```

Comment:
I can't find where do you use this?

Reply:
I added a test for fully custom cache structure (so we don't even reuse the _LocalCache). For sake of argument I based that structure on `cachetools`.